docs/DEV_ENV.md

```diff
@@ -52,19 +52,23 @@ Both the frontend and backend services will automatically reload when their sour
 
 To update frontend changes:
 
-1. add dependency to [src/frontend/package.json](src/frontend/package.json) (or add a new scripts command)
+1. add dependency to [src/frontend/package.json](../src/frontend/package.json) (or add a new scripts command)
 2. run `make local-update-frontend-deps` (updates package-lock.json)
 2. run `make local-sync`
 
 To update backend dependencies:
 
-1. add the dependency to [src/backend/Pipfile](src/backend/Pipfile)
+1. add the dependency to [src/backend/Pipfile](../src/backend/Pipfile)
```

Comment:
this file doesn't exist. should I point to a different one?

Reply:
Fixed it!